docs/source/aryn_cloud/aps_output.md

```diff
@@ -0,0 +1,97 @@
+# Output Format
+
+The output of the Aryn Partitioning Service is JSON.
+
+```text
+{ "status": in-progress updates,
+  "error": any errors encountered,
+  "elements": a list of elements }
+```
+
+## Element Format
+
+It is often useful to process different parts of a document separately. For example, you might want to process tables differently than text paragraphs, and typically small chunks of text are embedded separately for vector search. In the Aryn Partitioning Service, these chunks are called elements.
+
+Elements follow the following format:
+
+```text
+{"type": type of element,
+"bbox": Coordinates of bounding box around element,
+"properties": { "score": confidence score,
+                "page_number": page number element occurs on},
+"text_representation: for elements with associated text,
+"binary_representation: for Image elements when extract_table_structure is enabled }
+```
+
+An example element is given below:
+
+```json
+{
+    "type": "Text",
+    "bbox": [
+      0.10383546717026654,
+      0.31373721036044033,
+      0.8960905187270221,
+      0.39873851429332385
+    ],
+    "properties": {
+      "score": 0.9369918704032898,
+      "page_number": 1
+    },
+    "text_representation": "It is often useful to process different parts of a document separately. For example you\nmight want to process tables differently than text paragraphs, and typically small chunks\nof text are embedded separately for vector search. In the Aryn Partitioning Service, these\nchunks are called elements.\n"
+}
+```
+
+### Element Type
+
+```text
+"type": one of the element types below
+```
+
+|  Type   | Description  |
+| ------: | :----------- |
+| Title   |  Large Text  |
+| Text    | Regular Text | 
+| Caption | Description of an image or table |
+| Footnote | Small text found near the bottom of the page |
+| Formula | LaTeX or similar mathematical expression |
+| List-item | Part of a list |
+| Page-footer | Small text at bottom of page |
+| Page-header | Small text at top of page |
+| Image | A Picture or diagram. When `extract_images` is set to `true`, this element includes a `binary_representation` tag which contains a base64 encoded ppm image file. When `extract_images` is false, the bounding box of the Image is still returned. |
+| Section-header | Medium-sized text marking a section. |
+| Table | A grid of text. See the `extract_table_structure` option to extract information from the table rather than just detecting its presence. |
```

Comment:
isn't this returned as lower case "table" for some reason?

Reply:
Now changed in the docs to lowercase.